doc/styles/Google/Units.yml

```diff
@@ -5,4 +5,4 @@ nonword: true
 level: error
 tokens:
   - \d+(?:B|kB|MB|GB|TB)
-  - \d+(?:ns|ms|s|min|h|d)
+  - \d+(?:ns|ms|s|min|h)
```

Comment:
Out of curiosity, why were days removed here? Does it mess with "3d"?

Reply:
Yes. I either had to add in a bunch of exceptions to `accept.txt`, or just simplified the rule here.